include/opencv.h

```diff
@@ -0,0 +1,183 @@
+/*******************************************************
+ * Copyright (c) 2016, ArrayFire
+ * All rights reserved.
+ *
+ * This file is distributed under 3-clause BSD license.
+ * The complete license agreement can be obtained at:
+ * http://arrayfire.com/licenses/BSD-3-Clause
+ ********************************************************/
+#pragma once
+#include <af/array.h>
+#include <af/defines.h>
+#include <af/traits.hpp>
+#include <af/exception.h>
+#include <vector>
+#include <opencv2/core/core.hpp>
+
+namespace afcv
+{
+
+/*
+ * opencv -> arrayfire
+ */
+
+/**
+    \ingroup opencv_interop
+    @{
+*/
+
+/**
+   This function will copy data from a cv::Mat to an \ref af::array
+
+   \param[in]  input The data which will be loaded into the array
+   \param[in]  transpose flag specifying if the input should be transposed during copy
+
+   \returns an af::array with the contents of the OpenCv Mat
+*/
+static af::array array(const cv::Mat& input, bool transpose = true)
```

Comment:
Depending on the color model the client is using, we may need to convert to ArrayFire's RGB format.